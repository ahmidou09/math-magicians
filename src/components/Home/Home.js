import React from 'react';

function Home() {
  return (
    <div className="home-container">
      <h1>welcome to our page!</h1>
      <p>
        Lorem ipsum dolor sit amet consectetur adipisicing elit. Atque,
        a. Voluptates velit doloribus, dolor cumque commodi earum atque
        obcaecati cum maxime, non, saepe neque alias accusamus quae
        explicabo aut temporibus.
      </p>

      <p>
        Lorem ipsum dolor sit amet consectetur adipisicing elit. Atque,
        Voluptates velit doloribus, dolor cumque commodi earum atque
        obcaecati cum maxime, non, saepe neque alias accusamus quae
      </p>

    </div>
  );
}

export default Home;
