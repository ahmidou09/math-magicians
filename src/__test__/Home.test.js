/* eslint-disable linebreak-style */
import React from 'react';
import { render } from '@testing-library/react';
import Home from '../components/Home/Home';

test('renders Home component', () => {
  const { container } = render(<Home />);
  expect(container).toMatchSnapshot();
});
