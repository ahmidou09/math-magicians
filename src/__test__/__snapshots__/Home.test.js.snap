// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Home component 1`] = `
<div>
  <div
    class="home-container"
  >
    <h1>
      welcome to our page!
    </h1>
    <p>
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Atque, a. Voluptates velit doloribus, dolor cumque commodi earum atque obcaecati cum maxime, non, saepe neque alias accusamus quae explicabo aut temporibus.
    </p>
    <p>
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Atque, Voluptates velit doloribus, dolor cumque commodi earum atque obcaecati cum maxime, non, saepe neque alias accusamus quae
    </p>
  </div>
</div>
`;
